content/blog/2017/2017-01-02-jenkins-lifx-notifier-plugin.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Jenkins LIFX notifier plugin"
+tags:
+- plugins
+- lifx
+author: vgaidarji
+---
+
+[NOTE]
+====
+This is a link:http://vgaidarji.me/blog/2017/01/02/jenkins-lifx-notifier-plugin/[cross post] by link:https://github.com/vgaidarji[Veaceslav Gaidarji], open source developer and contributor to the Jenkins/Bitrise projects.
+====
+
+Some time ago I've encountered a link:http://www.lifx.com[LIFX] smart bulbs. These are the bulbs with a chip inside - 50% bulb, 50% chip. There're mobile applications for easy configuration and remote control of the bulb. Nothing special here, it simply works and is very convenient to have such bulbs in dormitory.
+
+=== Brilliant idea time
+
+[quote]
+____
+99% of ideas which come to our minds either were already implemented by someone else or they are shit.
+____
+
+And as it always happens, developer inside me generated an idea, which was implemented by someone else already.
+
+The idea was to `connect a LIFX bulb to Jenkins server and update bulb color according to a job state`.
+
+Before starting to work on such Jenkins plugin, I've searched for analogs in G and first links pointed me to existing Jenkins link:https://wiki.jenkins-ci.org/display/JENKINS/LIFX+notifier+plugin[plugin] and a link:https://www.cloudbees.com/blog/lifx-smart-lightbulb-build-notifier-plugin[blog post] from `Michael Neale`, author of this plugin, how he did it. Michael created a plugin which does exactly what I had in my mind.
+
+At this point I had to 2 options:
+
+* **forget** about the idea and give up
```

Comment:
I don't understand this option, if a plugin existed that met the original goal ("connect a LIFX bulb to a Jenkins server and update the color according to a job's state") then why would you give up on the idea?

Perhaps this meant "forget about building something new and just use the plugin"?